example/example-contract-originate-hello-world.ts

```diff
@@ -45,6 +45,7 @@ async function example() {
 
```

Comment:
When changing the name of a file, it should be changed in the scripts section of the package.json also: https://github.com/ecadlabs/taquito/blob/master/example/package.json#L7

Reply:
updated to
    "example:origination": "node -r ts-node/register --preserve-symlinks example-contract-originate-hello-world.ts",
    "example:origination2": "node -r ts-node/register --preserve-symlinks example-contract-originate-Ligo-Simple.ts",
    "example:origination3": "node -r ts-node/register --preserve-symlinks example-contract-originate-Ligo-vote.ts",